target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -132,7 +131,8 @@ d-team,newifi-d2)
 	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "$boardname:blue:wlan5g" "wlan1"
 	;;
 d-team,pbr-m1|\
-gehua,ghl-r-001)
+gehua,ghl-r-001|\
+hiwifi,hc5861b)
 	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "eth0.2"
```

Comment:
I cannot find "$boardname:blue:internet" being defined for hiwifi,hc5861b anywhere. Did I overlook it or is this a mistake?

Reply:
Should be a mistake. It's hc5661a which needs to be added here. fixed that in my staging tree...